extensions/amp-share-tracking/0.1/amp-share-tracking.js

```diff
@@ -163,4 +163,10 @@ export class AmpShareTracking extends AMP.BaseElement {
   }
 }
 
-AMP.registerElement('amp-share-tracking', AmpShareTracking);
+
+// Install the extension.
+// DO NOT SUBMIT: First, we must implement compiler rewrite path to unwrap
+// this code.
+AMP.extension('amp-share-tracking', () => {
```

Comment:
I think it'd be hard to add it dynamically, unless you want to wrap the whole thing. But that's also hard. With `AMP.push` we first get the complete concatenated binary before wrapping. This is not the case for development.